### PR TITLE
fix(notifier): serious logic issues with the file proxying thing

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Please remove the patch files

**Comment by roj1512** [comment]
Ik ik. I was just recovering.

File: notifier/main.ts [hunk 1]
```diff
@@ -46,13 +46,17 @@ bot.chatType(["group", "supergroup"]).filter((ctx) =>
     await ctx.reply("Could not resolve PR branch.");
     return;
   }
+  const { file_path } = await ctx.api.getFile(ctx.message.document.file_id);
```
Comment (KnorpelSenf) [inline notifier/main.ts:49]: What's wrong with using the files plugin here?

Comment (roj1512) [reply notifier/main.ts:49]: For downloading it?

Comment (KnorpelSenf) [reply notifier/main.ts:49]: For getting the URL

Comment (roj1512) [reply notifier/main.ts:49]: Make a suggestion

Comment (KnorpelSenf) [reply notifier/main.ts]: Did you find it?

Comment (KnorpelSenf) [reply notifier/main.ts]: There's `file.getUrl`

Comment (roj1512) [reply notifier/main.ts]: Oh, the URL. We can't use it for the URL.

Comment (KnorpelSenf) [reply notifier/main.ts]: Why not?

Comment (roj1512) [reply notifier/main.ts]: Because we don't use its URL here. We do in the `/file/:path` handler where we don't have grammY `ctx`.

Comment (KnorpelSenf) [reply notifier/main.ts]: Right

**Review by KnorpelSenf — APPROVED** [review]
LGTM

**Pull request merged**